.ci/check-format.sh

```diff
@@ -5,4 +5,9 @@ SOURCES=$(find $(git rev-parse --show-toplevel) | egrep "\.(cpp|h)\$" | egrep -v
 
 set -x
 
+for file in ${SOURCES};
+do
+    clang-format-6.0 ${file} > after-clang-format
```

Comment:
Rename `after-clang-format` to `expected-format` or something similar.

Reply:
I have replaced it with `expected-format`.